website/docs/getting-started/setup/installation-guides/restack.md

```diff
@@ -0,0 +1,28 @@
+# Restack
+
+ [Restack](https://www.restack.io) is platform to discover and deploy production ready open source tools in minutes.
+ You only need to bring your AWS account and within minutes, your favorite open source tool would be deployed on Kubernetes, on your own cluster.
+ Restack is made so that developers can concentrate on coding and don’t have to worry about DevOps. 
+ Restack provides auto scalable, cloud-native, instant, production-ready deployment of open source tools.
+
+ ## One-click deployment
+
+ You can deploy Appsmith on the Restack platform with just one click.
+
+ The easiest way to do this is
+
+ * [Login to Restack](https://restack.io/login) with your GitHub or email.
+ * Connect your AWS account to create a cluster. 
+ * Search for Appsmith in the restack console and click to trigger the deployment. 
+
+You could also follow the [detailed guide for deploying Appsmith available on Restack docs.](https://www.restack.io/docs/deploy-appsmith-on-restack#overview)  
```

Comment:
Let's not hyperlink so much text. Just the "guide" would suffice

And maybe rephrase it as "Follow the detailed [guide](https://www.restack.io/docs/deploy-appsmith-on-restack#overview) for deploying Appsmith available on Restack docs."

Reply:
The link to the guide also does not work. Let me check with Restack

Reply:
I've raised that the link is not working on the PR raised by Restack.